### ISSUE TITLE
feat(corset): possibility to refer to functions defined within a module in lookup files

### ISSUE THREAD

**Issue by lorenzogentile404** [Consensys/corset#141]
Using `(module module-name)` in the lookup file is not a clean solution (I am not sure if it even works) since lookups do not really belong to a module. Thus, it would be useful to refer to to functions defined within a module in lookup files in a way like `(module-name.function-name)`.

**Comment by DavePearce** [comment]
@lorenzogentile404 Just looking at this, and I have some queries.  The syntax for lookups is currently like e.g. this:

```lisp
(module m1)
(defcolumns X Y)
(deflookup test (m2.A m2.B) (X Y))

(module m2)
(defcolumns A B)
```

Where `X` and `Y` are implicitly in module `m1`.  We can also write the lookup outside of any module like e.g. this:

```lisp
(deflookup test (m2.A m2.B) (m1.X m1.Y))

(module m1)
(defcolumns X Y)

(module m2)
(defcolumns A B)
```

So, I'm not completely following what you're looking for here?

**Comment by DavePearce** [comment]
Something like this (which I could not get to work):

```
(module m1)
(defcolumns X Y)
(defpurefun (id x) x)

(module m2)
(defcolumns A B)

(module m3)
(deflookup test (m2.A m2.B) ((m1.id m1.X) m1.Y))
```

**Comment by delehef** [comment]
> (which I could not get to work):

Yes, for now explicitly qualified symbol lookups are restricted to columns only; it can be extended to functions in `tables.rs` if that's the desired behaviour.

**Comment by lorenzogentile404** [comment]
@DavePearce yes, your example clarifies the feature we would like to have.
